src/components/buttons/Button/Button.jsx

```diff
@@ -0,0 +1,110 @@
+import React from "react";
+import PropTypes from "prop-types";
+import { Icon } from "../../icons/Icon/Icon";
+
+import "./button.scss";
+
+/**
+ * Button
+ *
+ * All buttons used in the app
+ *
+ * @return {jsx}
+ */
+export const Button = ({
+  /* Add props here */
+  text,
+  iconName,
```

Comment:
@georgiganchev-7DIGIT Can we also add a property iconSize which will be used to provide the size of the icon.

Reply:
@georgiganchev-7DIGIT Also maybe iconColor